prowler/lib/check/check.py

```diff
@@ -201,11 +201,12 @@ def list_checks(provider: str) -> set():
     return sorted(available_checks)
 
 
-def list_categories(provider: str, bulk_checks_metadata: dict) -> set():
+def list_categories(bulk_checks_metadata: dict) -> set():
```

Comment:
Since there is no checks this will fail because you've removed one of the functions arguments.

Reply:
I removed it also when calling the function